README.md

```diff
@@ -7,7 +7,56 @@ This repository contains the Poste backend server application / API as a Python
 
 ---
 
-# Installation
+# Installation & Usage - Docker
+
+1. Clone this repository:
+
+`git clone git@github.com:Appyo-Poste/PosteBackend.git`
+
+2. Move into repository directory:
+
+`cd PosteBackend`
+
+3. Install and run Docker for your system: 
+
+>https://www.docker.com/get-started/
+
+4. Run the application via Docker using Docker Compose. Optional Make target provided for convenience.
+
+`docker compose up --build -d` or `make up` 
+
+5. The application will be running locally:
+ 
+>http://localhost
```

Comment:
```suggestion
>http://localhost:8000
```

Reply:
http://localhost/ is correct. It should be accessed by port 80 (and eventually 443 via https) so specifying the port is not appropriate